docs/en/03_How_tos/redirections.md

```diff
@@ -12,11 +12,20 @@ There's a few ways developers can implement redirections on their CWP stacks:
 
 Implementing in `.htaccess` is more performant than using PHP to redirect and allows more control than your standard `Director` redirection. If you are running your site with a [public/ webroot folder](https://docs.silverstripe.org/en/4/getting_started/directory_structure/), ensure that any rules are placed in `public/.htaccess`.
 
-The below example redirects redirection.com to www.redirection.com.
+Security Headers should be placed in `public/.htaccess`
```

Comment:
What are "Security Headers" - this is a new sentence that seems orphanded and doesn't have any examples.

Why should they not be in the root `.htaccess`? My understanding is that the root `.htaccess` gets evaluated first and then the public one, so that means the root file is the canonical one?

Reply:
I will add example for Security headers

Main job of root .`htaccess` is to redirect all incoming traffic to `public/.htaccess` which does the bulk of the work e.g. blocking access to some files, redirecting everything to `index.php`. 

If you place redirection rules in `public/.htaccess` you will redirects will end up with `/public/` in the redirected url

Reply:
Doesn't adding a `RewriteBase` solve that problem?

Reply:
```suggestion
Other HTTP headers, for example HTTP Security Headers, should be placed in `public/.htaccess` and not in `.htaccess`
```